### ISSUE TITLE
web-template folder is not included in pip package

### ISSUE THREAD

**Issue by gntech** [fragmuffin/cqparts#76]
When I try to use the `web_display`-functionality I get the following error

```

OSError: [Errno 2] No such file or directory: '/home/gustav/.local/lib/python2.7/site-packages/cqparts/display/web-template'

```

It seems the web-template folder is not included in the pip-package

**Comment by fragmuffin** [comment]
oh no!, it sure isn't!
That's interesting!, I thought I had a test for that.
Thanks for pointing that out!, that's a huge problem.

**Comment by fragmuffin** [comment]
fixed in #78, will be released in `v0.2.1` soon

Thanks again for the feedback @gntech :+1:
